openstack/elektra/alerts/secret.alerts

```diff
@@ -0,0 +1,28 @@
+groups:
+- name: secret.alerts
+  rules:
+  - alert: ElektraSecretAboutToExpire
+    expr: max by (namespace, secret) ((time() - kube_secret_created{namespace="elektra",secret=~"elektra-token|elektra-smtp"}) / 86400) >= 300
+    for: 5m
+    labels:
+      context: secret-rotation
+      playbook: docs/support/playbook/rotate-local-kubernetes-secret
+      service: elektra
+      severity: info
+      support_group: containers
+    annotations:
+      summary: 'PostgreSQL user secret will expire soon'
+      description: 'The secret {{`{{`}} $labels.namespace {{`}}/{{`}} $labels.secret {{`}}`}} is nearly 365 days old. Please rotate it soon by following the attached playbook.'
+
+  - alert: ElektraSecretAboutToExpire
+    expr: max by (namespace, secret) ((time() - kube_secret_created{namespace="elektra",secret=~"elektra-token|elektra-smtp"}) / 86400) >= 330
+    for: 5m
+    labels:
+      context: secret-rotation
+      playbook: docs/support/playbook/rotate-local-kubernetes-secret
+      service: elektra
+      severity: warning
+      support_group: containers
+    annotations:
+      summary: 'PostgreSQL user secret is expired'
+      description: 'The secret {{`{{`}} $labels.namespace {{`}}/{{`}} $labels.secret {{`}}`}} is over 365 days old. Please rotate it as soon as possible by following the attached playbook.'
```

Comment:
Durations named in the descriptions do not match the thresholds in the alert expressions.